.github/workflows/build.yml

```diff
@@ -0,0 +1,120 @@
+#
+#  Copyright (c) 2022, The OpenThread Authors.
```

Comment:
Is this the correct copyright?

Reply:
I believe so. I think this would be considered a redistribution. We also maintain the OpenThread copyright on the scripts in `script/` copied from `ot-efr32`. See https://github.com/SiliconLabs/KNX-IOT-STACK/pull/1/files#diff-99b83631df533ea1d9c97396a043a4670464f523d0c1e60b0a1cab6dddd440f9